src/main/java/com/assure/movie/service/MemberService.java

```diff
@@ -32,6 +34,9 @@ public Member getMember(Long id) throws NotFoundErrorException {
                 new NotFoundErrorException("Not found member with id: "+ id)
```

Comment:
This comment should be in like 32: Is it necessary to declare the throws _NotFoundErrorException_? I see this exceptions extends from _RuntimeException_.

Reply:
The runtime errors do not to be thrown,  so my declaration of **throws** is unnecessary.